cmd/main.go

```diff
@@ -130,7 +130,9 @@ func main() {
 	}
 
 	// Start the security policy controller.
-	StartSecurityPolicyController(mgr, commonService)
+	if cf.CoeConfig.EnableVPCNetwork && commonService.NSXClient.NSXCheckVersionForSecurityPolicy() {
```

Comment:
If EnableVPCNetwork is true, you need to check NSX version(>= 4.1.1). If NSX version is lower than 4.1.1, we cannot start VPC related controllers

Reply:
Yes, we should complete the featureSupported func later, and apply this pattern to start every controller.
This is just a pattern here.